test/e2e/stack.go

```diff
@@ -90,11 +99,21 @@ func (e *e2eCfnStack) deployStack(ctx context.Context, logger logr.Logger) error
 		},
 	}
 
+	var buf bytes.Buffer
+	cfnTemplate, err := template.New("hybridCfnTemplate").Parse(cfnTemplateBody)
+	if err != nil {
+		return fmt.Errorf("Failed to parse hybrid-cfn.yaml template: %v", err)
```

Comment:
nit: avoid using words like failed or error in an error message, it's redundant into. also error message should not be capitalized. also use %w for error wrapping
```suggestion
		return fmt.Errorf("parsing hybrid-cfn.yaml template: %w", err)
```